modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -108,6 +108,7 @@ inline IE::Precision toIE(int depth) {
     case CV_8U:  return IE::Precision::U8;
     case CV_32S: return IE::Precision::I32;
     case CV_32F: return IE::Precision::FP32;
+    case CV_16F: return IE::Precision::FP16;
```

Comment:
@alalek Could you explain what is the `CV_16F` actually mean ? 
I'm a bit confused, because `cv::convertFp16` function returns `CV_16S` depth.

Reply:
Discussed locally, thanks!

Reply:
Please share this info here for the history.

Reply:
Dedicated `cv::convertFp16` call is deprecated and is going to be replaced with common `.convertTo()` call.